### PR TITLE
[Metricbeat] Change Account ID to Project ID in `gcp.billing` module

### LINKED ISSUE #26357: GCP billing reports wrong account ID
Thank you @fxdgear for pointing out that the account ID in events reported by `gcp.billing` is not the actual account ID. It is using project ID instead.

### PR DESCRIPTION
**Pull request by legoguy1000**
## What does this PR do?

Currently the GCP Project ID is being ingested as `cloud.account.id` instead of `cloud.project.id`.  This PR fixes that.

## Why is it important?

See above

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## Related issues

- Closes #26357

### CODE REVIEW

**Review by kaiyan-sheng — COMMENTED** [review]
Hi @legoguy1000 thank you for your contribution!! Do you know if we can get the account ID and account name in GCP from the given project ID? I would like to report both account ID/Name and project ID in `gcp.billing`.

**Comment by legoguy1000** [comment]
@kaiyan-sheng based on this, https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables#data-schema,  I made additional changes to the query to pull the account info.  I don't have a GCP account with any actual billing so I don't have a good way to validate the query besides just looking at the schema in the above link.

**Comment by legoguy1000** [comment]
@kaiyan-sheng can you run the tests when you get a chance.

**Review by kaiyan-sheng — COMMENTED** [review]
Thanks for the change. I tested it locally and here is what I saw:
```
2021-06-23T15:33:35.456-0600    ERROR   [billing]       billing/billing.go:242  bigquery Wait failed: googleapi: Error 400: SELECT list expression references column billing_account_id which is neither grouped nor aggregated at [6:33], invalidQuery

2021-06-23T15:54:33.736-0600    ERROR   [billing]       billing/billing.go:241  bigquery Wait failed: googleapi: Error 400: SELECT list expression references column cost_type which is neither grouped nor aggregated at [6:33], invalidQuery
```
I will look into it a bit more tomorrow :)

File: x-pack/metricbeat/module/gcp/billing/billing.go [hunk 1]
```diff
@@ -261,26 +263,29 @@ func (m *MetricSet) queryBigQuery(ctx context.Context, client *bigquery.Client,
...
+func createEvents(rowItems []bigquery.Value, projectID string) mb.Event {
 	event := mb.Event{}
 	event.MetricSetFields = common.MapStr{
-		"invoice_month": rowItems[0],
-		"project_id":    rowItems[1],
-		"cost_type":     rowItems[2],
-		"total":         rowItems[3],
+		"invoice_month":      rowItems[0],
+		"project_id":         rowItems[1],
+		"project_name":       rowItems[3],
```
Comment (legoguy1000) [inline x-pack/metricbeat/module/gcp/billing/billing.go:278]: Somehow I forgot how to count. I'll fix this in the morning.

Comment (legoguy1000) [reply x-pack/metricbeat/module/gcp/billing/billing.go]: done

**Review by kaiyan-sheng — COMMENTED** [review]
OK, adding `GROUP BY` and `ORDER BY` fixed the issue:
```
query := fmt.Sprintf(`
			SELECT
				invoice.month,
				project.id,
				project.name,
				billing_account_id,
				cost_type,
			  (SUM(CAST(cost * 1000000 AS int64))
				+ SUM(IFNULL((SELECT SUM(CAST(c.amount * 1000000 as int64)) FROM UNNEST(credits) c), 0))) / 1000000
				AS total_exact
			FROM %s
			WHERE project.id IS NOT NULL
			AND invoice.month = '%s'
			AND cost_type = '%s'
			GROUP BY 1, 2, 3, 4, 5
			ORDER BY 1 ASC, 2 ASC, 3 ASC, 4 ASC, 5 ASC;`, tableMeta.tableFullID, month, costType)
```

**Comment by legoguy1000** [comment]
@kaiyan-sheng Query fixed.  Should be good for CI tests now.

**Comment by legoguy1000** [comment]
@kaiyan-sheng Looks good to merge now.

**Review by kaiyan-sheng — APPROVED** [review]

**Pull request merged**